packages/botonic-react/src/components/button.jsx

```diff
@@ -38,8 +38,8 @@ export const Button = props => {
     else if (props.payload) sendPayload(props.payload)
     else if (props.url) {
       window.open(props.url)
-      props.onClick()
-    } else if (props.onClick) props.onClick()
+    }
+    if (props.onClick) props.onClick()
```

Comment:
another difference is that now props.onClick() is also called when there's path or payload

Reply:
yes because despite of being a path or payload we can want to call a method